server/src/app/GaWorker.ts

```diff
@@ -45,15 +45,28 @@ export class GaWorker {
    */
 
   initialBots(): Bot[] {
-    return [
-      "C4,G4,G4,F4",
-      "A4,B4,G4,E4",
-      "G4,B4,D5,E5",
-      "E4,E4,E4,A4",
-      "B4,A4,G4,D4",
-      "D4,G4,C5,D5",
-      "E4,G4,E4,D4"
-    ].map((str) => new Bot(0, Melody.fromString(str)));
+    let random_initial = false;
```

Comment:
Small thing, but I would make this an argument that can be passed into the `initialBots` function :) . This will make it easier to do the tests we had in mind without needed to rebuild the code.

Reply:
Also camel casing would be nice `randomInitial` :)